### ISSUE TITLE
Custom element does not exist (Fire tablet)

### ISSUE THREAD

**Issue by rysm83** [dermotduffy/frigate-hass-card#895]
I have added the frigate card to my dashboard and works perfectly  fine in the PC, Iphone and Fire tablet 10.  However I can not get this to work on the lower models  of fire tablets.  I have a fire 8 tablet that I am using for display.  No matter which I use Fully kiosk browser, Native Home assistant app or web page I get the same. 
Custom element does not exist: Frigate Card error message with red box.
I have cleared the cache in all browsers and rebooted nothing seems to work.  Is this something related to hardware capabilities?

**Comment by atv2016** [comment]
i have the same on a samsung family hub tv (probably use webkit or something)

**Comment by spacebares** [comment]
Have the same problem on bluestacks 5.

**Comment by felipecrs** [comment]
> No matter which I use Fully kiosk browser, Native Home assistant app or web page I get the same.

Which specific Browser have you tried? Which version? I would suggest trying on Chrome 110+.

**Comment by dermotduffy** [comment]
I played around on Browserstack and found I can reproduce this issue if I use _very_ old versions of browsers (e.g. Chrome 70), so the issue is likely Javascript-based incompatibilities from old browsers or old hardware with old webview. I don't think supporting such hardware/browsers is the right use of the development cycles available, so I'm going to close this bug. I've added a browser support note in https://github.com/dermotduffy/frigate-hass-card/pull/1036 . I realize this is not a terribly satisfying answer, but I think it's a realistic one...
